client/src/rpc_request.rs

```diff
@@ -107,6 +108,16 @@ pub enum RpcError {
     ForUser(String), /* "direct-to-user message" */
 }
 
+// Extensible struct to pass config fields to rpc
+// As long as new fields are added as `Option`s, this will remain backwards compatible
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RpcConfig {
```

Comment:
I was thinking it might be useful to have a generic config struct, but since this is the only caller of it, I'll make it specific for now, and rework entirely once we have a use case for generic configs.